apps/condo/domains/news/schema/NewsItem.js

```diff
@@ -153,6 +153,40 @@ const NewsItem = new GQLListSchema('NewsItem', {
             },
         },
 
+        compactScopes: {
+            schemaDoc: 'Returns the number of scopes that are specified for sending the news, and also the first two of them.\n' +
+                'Used to reduce requests for get of scopes in the UI',
+            type: 'Virtual',
+            extendGraphQLTypes: ['type ShortScopesField { count: Int!, firstTwo: [NewsItemScope]! }'],
```

Comment:
Maybe not use array size as a part of the name of the variable?
```suggestion
            extendGraphQLTypes: ['type ShortScopesField { count: Int!, firstOnes: [NewsItemScope]! }'],
```